src/goto-instrument/cover.cpp

```diff
@@ -1375,9 +1390,10 @@ void instrument_cover_goals(
   Forall_goto_functions(f_it, goto_functions)
   {
     if(f_it->first==ID__start ||
-       f_it->first=="__CPROVER_initialize")
+       f_it->first=="__CPROVER_initialize" ||
```

Comment:
Use `CPROVER_PREFIX` "initialize" (and `#include <util/cprover_prefix.h>`) [yes, there are ~2000 other locations where that should get fixed]